README.md

```diff
@@ -47,6 +47,33 @@ worker.post('/event/*', analyticsMiddleware);
 More than one middleware can be registered with one worker. You just need to keep in mind that the `request` and `response` objects will be passed to each middleware in the same order that they were registered.
 
 
+## Specifying routes
+
+While registering your middlewares, you are not limited just to predefined paths, you have the choice to specify your routes by using *placeholders*. These placeholders are loosely based on [Express' path strings](http://expressjs.com/guide/routing.html#route-paths).
```

Comment:
I want a second opinion about the use of the name _placeholder_ for this feature. I got the point, I like it but I preffer to hear from @arcturus as well.